app/src/main/kotlin/model/io/neo4j/Neo4jRepository.kt

```diff
@@ -0,0 +1,182 @@
+package model.io.neo4j
+
+import model.graphs.DirectedGraph
+import model.graphs.UndirectedGraph
+import model.graphs.WeightedDirectedGraph
+import model.graphs.WeightedUndirectedGraph
+import model.graphs.abstractGraph.Graph
+import model.graphs.abstractGraph.Vertex
+import org.neo4j.driver.*
+import java.io.Closeable
+
+const val DIR_LABEL = "POINTS_TO"
+const val UNDIR_LABEL = "CONNECTED_TO"
+
+class Neo4jRepository(uri: String, user: String, password: String) : Closeable {
+    private val driver = GraphDatabase.driver(uri, AuthTokens.basic(user, password))
+    private val session = driver.session()
+
+    fun getGraphNames(): List<String> {
+        val result = session.executeRead { tx ->
+            tx.run(
+                "MATCH (v) " +
+                "RETURN " +
+                "distinct labels(v) AS label"
+            ).list()
+        }
+        println(result)
```

Comment:
💡 If you wanted to do logging, there are special libraries for that. You even have some of them imported in Gradle (`org.slf4j:slf4j-api:1.7.36`, `ch.qos.logback:logback-classic:1.4.12`) but I don't think I've seen you using them anywhere.